help/access/file-shares.md

```diff
@@ -1,302 +1,237 @@
 
-   In the identity/email field, start typing a person’s name, username, email, or group name. Pick from the list of the
-   matching identities.
+When satisfied with the data, click **Done** to open the dataframe in Datagrok. From there, you can export it, make further edits, save it as a project, or share with others via URL.
 
-3. From the respective dropdowns, select access privileges for either or both: (1) the connection and (2) individual
-   files/folders. You can select any or all of the following options<!--TBU-->:
+![Text Manager](text-manager.gif)
 
-   * _Can view_: Users can view, open, and download
-   * _Can edit_: Users can rename and edit
-   * _Can delete_: Users can delete
-   * _Can share_: Users can reshare with any other user or group.<!--how does it work with URL links? -->
+## File Manager
 
-   > Note: For each _file share_, Datagrok automatically assigns a _friendly name_ (a name displayed in the UI) that
-   > specifies all the directory names starting from the root folder. While you _can_ change the _friendly name_, we
-   > encourage you to exercise caution. Changing the item's _friendly name_ also changes its unique
-   > _namespace-qualified name_ used in URL links, automation workflows, and more.
+The **File Manager** is an interface that allows you to manage connections, browse and preview file content, and perform standard file and folder actions such as opening, downloading, deleting, and renaming. To access an object's context actions, right-click it or left-click and expand the **Actions** pane in the **Context Panel** on the left. By clicking a file or folder in the **File Manager**, you can open its preview. Double-clicking a file opens it in Datagrok, and double-clicking a folder expands its content.
 
-4. Optionally, enter a description in the text field provided. You may also notify the users you share with. If you
-   don’t want to send a notification, uncheck the **Send notification** checkbox.
+:::note
 
-   > Note: To send an email notification, enter the user's email in the identity/email field. The email notification
-   > contains a link to the shared item and entered description. If you enter a user or group name, they will be
-   > notified via the Datagrok interface.
+If you don't see a certain action, it may be due to insufficient permissions. For files and folders shared with you, contact the credentials owner. If you are a credentials owner, contact the data source owner.
 
-5. Click **OK** to share. Once shared, the shared item appears in the recipient's **Folder Tree**.
+:::
 
-   ![Share a folder](share-the-folder.gif)
+In addition to the hierarchical browsing, the **File Manager** offers an array of advanced preview and data augmentation capabilities using **Directory**, **Preview**, and **Context Panel**.
```

Comment:
Too long. Drop "an array of". Still does not read well.

I would say "File Manager consists of the Tree, Directory, Preview, and Context Panel blocks"